sdk/python/tests/integration/registration/test_inference.py

```diff
@@ -138,6 +138,26 @@ def test_update_data_sources_with_inferred_event_timestamp_col(simple_dataset_1)
             )
 
 
+@pytest.mark.integration
+@pytest.mark.universal
+def test_update_data_sources_with_inferred_event_timestamp_col(universal_data_sources):
+    (_, _, data_sources) = universal_data_sources
+
+    # remove defined event_timestamp_column to allow for inference
+    for data_source in data_sources.values():
```

Comment:
btw should we take data sources and make a deep copy? as is this will mutate the sources and impact other tests potentially

Reply:
hmm is it not a fixture? I thought it gets setup and torn down for every test?

Also, I'm looking at the failed linter and can't see what the issue is

Reply:
they're session scoped though so should be reused across tests (at least that was the intention)

Reply:
did you try running make format-python?